src/main/resources/application.yaml

```diff
@@ -49,5 +56,13 @@ oidc:
 ccd:
   data-store:
     host: ${CCD_DATA_STORE_HOST:http://localhost:4452}
+    whitelisted-urls: /ccd/searcCases\?ctid.*
```

Comment:
we should use an env var as a value here, similar to some other properties. So that we can change it just by doing  a flux PR on Prod without having to release MCA

Reply:
Adding **${MANAGE_CASE_DS_PROXY_URLS_WHITELIST:/ccd/searchCases.*}**

Reply:
can we call it  ${MCA_DS_PROXY_URLS_WHITELIST:/ccd/searchCases.*} please?